### PR TITLE
spotinfo 0.4.1

### PR DESCRIPTION
**Pull request by alexei-led**
Signed-off-by: Alexei Ledenev <alexei.led@gmail.com>

- [v] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [v] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [v] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [v] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [v] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/spotinfo.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  license "Apache-2.0"
+  head "https://github.com/alexei-led/spotinfo.git"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+  depends_on "make" => :build
```
Comment (SMillerDev) [inline Formula/spotinfo.rb:15]: Doesn't it work with the macOS version?

**Comment by nandahkrishna** [comment]
Audit failure:
```
spotinfo:
  * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
```

**Comment by nandahkrishna** [comment]
Thank you for your PR, @alexei-led. Unfortunately, the formula currently does not meet the requirement for inclusion in `homebrew/core`:
```
spotinfo:
  * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
```
You are welcome to submit this again when this requirement is satisfied.

If you wish, you can host this formula in your own tap in the meantime. Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

File: Formula/spotinfo.rb (hunk 1 above)
Comment (alexei-led) [reply Formula/spotinfo.rb:15]: I develop it on Mac. It works on Mac/Linux/Windows

**Pull request closed without merging**